hack/run-origin-tests.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
+
+skopeo_workdir=$(mktemp -d)
+function finish_skopeo {
+  rm -rf ${skopeo_workdir}
+}
+trap finish_skopeo EXIT
+
+function get_ose_tests_binary {
+    # When running in a pod, we can't directly use the image.
+    # For this reason, we download the image and fetch the binary from the right layer
+    skopeo copy docker://"$ORIGIN_TESTS_IMAGE" oci:${skopeo_workdir}
+    echo "Fetching openshift-tests binary from $ORIGIN_TESTS_IMAGE"
+    for layer in ${skopeo_workdir}/blobs/sha256/*; do
+        echo "layer=${layer}"
+        set +e
+        testsbin=$(tar -t -f "$layer" | grep openshift-tests)
+        set -e
+        if [[ $testsbin ]]; then
+            echo "Found $testsbin on $layer"
+            tar xfv "$layer" "$testsbin"
+            mv "$testsbin" _cache/tools/openshift-tests
+            chmod +x _cache/tools/openshift-tests
+            rm -rf ${skopeo_workdir}
+            break
+        fi
+    done
+}
+
+CONTAINER_MGMT_CLI="${CONTAINER_MGMT_CLI:-docker}"
+TESTS_REPORTS_PATH="${TESTS_REPORTS_PATH:-/tmp/artifacts/}"
+
+ORIGIN_TESTS_IMAGE="${ORIGIN_TESTS_IMAGE:-quay.io/openshift/origin-tests:4.8}"
+ORIGIN_TESTS_FILTER="${ORIGIN_TESTS_FILTER:-openshift/conformance/parallel}"
+TEST_PROVIDER="${TEST_PROVIDER:-}"
+
+failed=false
+
+echo "Running $ORIGIN_TESTS_FILTER tests"
+
+mkdir -p "$TESTS_REPORTS_PATH"
+
+mkdir -p _cache/
+cp -f "$KUBECONFIG" _cache/kubeconfig
+
+# The cluster infrastructure provider, will automatically default to the correct value.
```

Comment:
what's a provider?

Reply:
the cluster type, i will update the comment